.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
```

Comment:
Idea: you could add a `package-ecosystem: gomod` configuration as well, so that dependencies like `ejson-1.3.3` get the automatic PR treatment as well.